README.md

```diff
@@ -1 +1,7 @@
-# examples
\ No newline at end of file
+# Useful Roc Examples
```

Comment:
I've changed the title, I don't think packages should be put here in the long term.
Obviously the examples will be applications with an app header but I would prefer not to put applications in the title, it makes it sound like a fibonacci example would not belong here.

Reply:
@Anton-4 what do you think about pulling the packages out and just using a URL? I can put a release on my package-explorations repository and just use that for now. My thoughts were this would show people an example for how to make a package. Maybe we could add notes to README just explaining that these packages are for demonstration purposes only, and are not standard library or anything well supported.

Reply:
Yes, let's use URLs for packages in general and perhaps one using a local folder to show how that works.
The disclaimer in the README sounds good :+1: